packages/button/src/LionButton.js

```diff
@@ -213,14 +203,9 @@ export class LionButton extends DisabledWithTabIndexMixin(SlotMixin(LitElement))
    * @param {Event} e
    */
   __clickDelegationHandler(e) {
-    if ((this.type === 'submit' || this.type === 'reset') && e.target === this) {
```

Comment:
Now I am really questioning why we had this logic here in the first place. I doubt it was without reason. Could you check if implicit form submission works in IE11 with this? We currently don't run the tests automatically in IE11 so we have to kinda check it manually.

Reply:
It was because of a double dispatch. But with stopImmediatePropagation we avoit to dispatch host one and only dispatch the native one. If you are listening the event in the lion-button it will be the currentTarget and the target will be the native button.

Reply:
It works!

[image](https://user-images.githubusercontent.com/948953/95856645-a0beab00-0d5a-11eb-80c8-799bfda706da.png)